rst2pdf/tests/input/test_issue_239_2.txt

```diff
@@ -1,3 +0,0 @@
-.. |warning| image:: warning.svg
```

Comment:
Did we lose all SVG support with losing inkscape?

Reply:
No. SVG support remains supported vis [Svglib](https://pypi.org/project/svglib/).

Reply:
But svglib will not provide support for gradients ...